torch/distributed/rpc/api.py

```diff
@@ -28,7 +28,7 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
-def join_rpc():
+def wait_all_workers():
     r"""
     Block until all local and remote RPC processes reach this method, process
```

Comment:
Can we expand the docstring here to clearly indicate that this method should be used as part of termination and there is no guarantee that the RPC framework will work after this method returns. Would be also helpful to have an example here of how to just `wait_all_workers` and `shutdown` together.